Public/Src/Engine/Dll/Tracing/LogEventId.cs

```diff
@@ -216,7 +216,7 @@ public enum LogEventId : ushort
         FailedToAcquireDirectoryLock = 7116,
         UsingRedirectedUserProfile = 7117,
         FailedToRedirectUserProfile = 7118,
-        ResourceBasedCancellationIsEnabledWithSharedOpaquesPresent = 7119,
+        // ResourceBasedCancellationIsEnabledWithSharedOpaquesPresent = 7119,

```

Comment:
>ResourceBasedCancellationIsEnabledWithSharedOpaquesPresent [](start = 11, length = 58)

Remove?

Reply:
Whats our common practice here? A few lines above we have commented out ones and some that say // XXX was YYYY. Do we need to keep any traces around?

Reply:
```suggestion
        // was: ResourceBasedCancellationIsEnabledWithSharedOpaquesPresent = 7119,
```

Reply:
I usually put a "was" comment in there so if it comes up with people running old versions of BuildXL we can still see what it meant in the past without doing too much research. Then when the line is sufficiently old we can also recycle the ID